### PR TITLE
[ADD] AFC-1486a Send Joint Planning: Add outplacement_joint_planning

### PR DESCRIPTION
**Pull request by n-development**

### CODE REVIEW

File: outplacement_completion_report_ipf_client/models/client_config.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def get_url(self, path):
             url = self.url + '/' + path
         return url
 
-    def post_report(self):
+    def post_report(self, ):
```
Comment (arturchilangwa-vertel) [inline outplacement_completion_report_ipf_client/models/client_config.py]: @AnatoliiYakymets  
Is this change necessary?

**Comment by arturchilangwa-vertel** [comment]
Installed `outplacement_joint_planning` worked! Code for it looks good also, I just have a questioning about https://github.com/vertelab/odoo-outplacement/pull/6#discussion_r541027075

File: outplacement_completion_report_ipf_client/models/client_config.py (hunk 1 above)
Comment (n-development) [reply outplacement_completion_report_ipf_client/models/client_config.py]: Hello @anderswallenquist, This's the mistake because I wanted to change it to implementation the ticket AFC 1486c.

**Review by timforsstromAF — COMMENTED** [review]
I dont understand why there is data (John Doe) in this?

**Review by timforsstromAF — APPROVED** [review]

**Review by timforsstromAF — COMMENTED** [review]
Is it test data or what is the purpose of "John Doe"
